commands/remove.rb

```diff
@@ -18,7 +39,12 @@ def self.remove(pkg, verbose)
     # their dependencies, as those are needed for ruby and crew to run,
     # and thus should not be removed.
     # essential_deps = recursive_deps(CREW_ESSENTIAL_PACKAGES)
-    essential_deps = device_json[:essential_deps]
+    if device_json[:essential_deps].blank?
```

Comment:
is there a situation in which this will be empty? it'll be nil if the field isnt in device.json, which does happen and is the cause of the issue, but I can't think of a scenario in which it'll be empty.

Reply:
I'm just trying to handle the nil error... .`nil?` is complaining... but that could be a race condition from having older code in the mix. I'm just opening the M121 container for testing...

Reply:
Fixed the root cause.